src/DevSoc.Eventathon/ClientApp/src/api/events-endpoint.js

```diff
@@ -0,0 +1,27 @@
+import axios from 'axios';
+import useSWR from 'swr';
+
+export const useGetEvent= (id) => {
```

Comment:
minor thing, but this could just be `useEvent(id)` and `useEvents()` (the `Get` here is redundant)

Reply:
Cheers. I'll neaten that up